modules/data-entry/src/main/frontend/src/dataHomepage/FormView.jsx

```diff
@@ -49,6 +49,7 @@ function FormView(props) {
   const [ subtitle, setSubtitle ] = useState();
   const [ questionnairePath, setQuestionnairePath ] = useState();
   const [ qFetchSent, setQFetchStatus ] = useState(false);
+  const [ filtersJsonString, setFiltersJsonString ] = useState(expanded ? location.hash.split(/&filters=(.+)/)[1] || "" : "");
```

Comment:
This is fragile, we may add other things after filters in the future. To be more flexible, I would use something like:

```suggestion
  const [ filtersJsonString, setFiltersJsonString ] = useState(expanded ? new URLSearchParams(window.location.hash.substring(1)).get("filters") : "");
```

But we could go further, and use different parameter names for different filters. Here we could identify that these filters apply to the forms table with:

```suggestion
  const [ filtersJsonString, setFiltersJsonString ] = useState(expanded ? new URLSearchParams(window.location.hash.substring(1)).get("forms:filters") : "");
```

And we could also apply the filters when loading the dashboard, to allow sharing a link to the dashboard with the current filters applied:

```suggestion
  const [ filtersJsonString, setFiltersJsonString ] = useState(new URLSearchParams(window.location.hash.substring(1)).get("forms:filters"));
```